sale_order_remove_portal_timesheet/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Sale Order Remove Portal Timesheet",
+    "summary": """
+        Sale Order Remove Portal Timesheet
```

Comment:
Instead of this, a description like "Sale Order - Removes timesheet access from Portal" should be better (current `summary` does not change from `name`)